ci/security-scan/weekly_commit_ids.env

```diff
@@ -1,30 +1,30 @@
 odh-minimal-notebook-image-main=quay.io/opendatahub/workbench-images@sha256:e9d6a6ee0e1ce6878d3a00dd9ffe85ffb536298ce1fe4d9c577ba0159c69a7f3
-odh-minimal-notebook-image-n=quay.io/opendatahub/workbench-images@sha256:cde20ac445d25c70d95042a546334c398ed3fca73e85530f0ffef3cbdb6ec746
-odh-minimal-notebook-image-n-1=quay.io/opendatahub/workbench-images@sha256:9323e689ec6ab1abb3cbdfd6258811bd57376c6b3e48f71838408cbb0b8b24a3
+odh-workbench-jupyter-minimal-cpu-py311-ubi9-n=quay.io/opendatahub/workbench-images@sha256:cde20ac445d25c70d95042a546334c398ed3fca73e85530f0ffef3cbdb6ec746
+odh-workbench-jupyter-minimal-cpu-py311-ubi9-n-1=quay.io/opendatahub/workbench-images@sha256:9323e689ec6ab1abb3cbdfd6258811bd57376c6b3e48f71838408cbb0b8b24a3
```

Comment:
As we don't support the n-1 should we include these on the weekly scan?

Reply:
not relevant to this issue, lets track this separately and fix it later.

Reply:
thnx @jiridanek for creating the tracker on odh,i will addit here for reference